src/home/user-record-util.c

```diff
@@ -1404,6 +1404,208 @@ int user_record_is_supported(UserRecord *hr, sd_bus_error *error) {
         return 0;
 }
 
+static int remove_safe_json_fields_regular(JsonVariant **v) {
+        _cleanup_(json_variant_unrefp) JsonVariant *blobs = NULL;
+        int r;
+
+        assert(v);
+
+        char **safe_fields = STRV_MAKE(
+                /* For display purposes */
+                "realName",
+                "emailAddress", /* Just the $EMAIL env var */
+                "iconName",
+                "location",
+
+                /* Basic account settings */
+                "shell",
+                "umask",
+                "environment",
+                "timeZone",
+                "preferredLanguage",
+                "additionalLanguages",
+
+                /* Authentication methods */
+                "pkcs11TokenUri",
+                "fido2HmacCredential",
+                "recoveryKeyType",
+
+                "lastChangeUSec", /* Necessary to be able to change record at all */
+                "lastPasswordChangeUSec" /* Ditto, but for authentication methods */
+        );
+        char **safe_blobs = STRV_MAKE(
+                /* For display purposes */
+                "avatar",
+                "login-background"
```

Comment:
please suffix with comma always

Reply:
It doesn't compile if you do, because of STRV_MAKE